src/state/slices/opportunitiesSlice/constants.ts

```diff
@@ -137,3 +137,9 @@ export const DEFI_PROVIDER_TO_METADATA: Record<DefiProvider, DefiProviderMetadat
     url: 'https://app.shapeshift.com',
   },
 }
+
+export const isRunePoolOpportunity = (
```

Comment:
```suggestion
export const isRunePoolUserStakingOpportunity = (
```

Reply:
Changed in [70aeeab](https://github.com/shapeshift/web/pull/7472/commits/70aeeab7af3bc9f83c432fe26f48cfddce87ce27)